### ISSUE TITLE
Site/App is not working on Oppo Find X2 Pro

### ISSUE THREAD

**Issue by SlothDaTurquoise** [RobinLinus/snapdrop#395]
Hi, Oppo find X2 Pro is my daily phone. When I discovered snapdrop I loved it however that is not working with my device at all. I tried everything. Many types of browser etc. I tried connected to my PC with my brother's Samsung galaxy a40 and everything is working just fine. 
Snapdrop see Oppo on my PC and I see PC on my phone too. When I'm trying to send something photo, video or even a message just nothing happens. It back to the "desktop" I mean main screen the animation of floating signal is frozen and nothing happens. 
The problem is active both when I'm using app or site. 
I dunno does it only my device or all Oppo Find X2 Pro devices has that problem too, I can't find any info about that on web.
Thanks for any feedback

**Comment by dtantono** [comment]
Yeah, I use Android Tablet and It detected my devices. But, it cannot send anything to my PC browser.

**Comment by hassan-sohail12** [comment]
Ok contact me i solve this problem
